client/blocks/login/style.scss

```diff
@@ -20,9 +22,6 @@ $breakpoint-mobile: 782px; //Mobile size.
 				box-shadow: 0 0 0 2px var(--studio-jetpack-green-30);
 			}
 		}
-		.button, .components-button, input[type="text"] {
-			border-radius: 4px;
```

Comment:
I see the [reason to add this](https://github.com/Automattic/wp-calypso/pull/102877#pullrequestreview-2810654675) was for consistency with .com, so it's no longer needed